packages/jsii/lib/validator.ts

```diff
@@ -70,6 +71,19 @@ function _defaultValidations(): ValidationFunction[] {
         }
     }
 
+    function _enumMembersMustUserUpperSnakeCase(_: Validator, assembly: spec.Assembly, diagnostic: DiagnosticEmitter) {
+        for (const type of _allTypes(assembly)) {
+            if (!spec.isEnumType(type)) { continue; }
+
+            for (const member of type.members) {
+                if (member.name && member.name !== Case.constant(member.name)) {
+                    diagnostic(ts.DiagnosticCategory.Error,
+                               `Enum members must use TRUMP_CASE: ${member.name}`);
```

Comment:
I wonder if "ALL_CAPS" is a more suitable name

Reply:
I actually like it more, will make the change before pushing

Reply:
Done.